frontend/src/lib/utils.tsx

```diff
@@ -219,6 +219,11 @@ export const selectorOperatorMap: Record<string, string> = {
     is_not: "≠ doesn't equal",
 }
 
+export const cohortOperatorMap: Record<string, string> = {
```

Comment:
something like `user belongs to this cohort`/`user doesn't belong to this cohort`, might make it easier for users to understand what the selector does. Though will wait for Michael's input on this

Reply:
Yeah, the problem is how the operator component is structured on the frontend - the operator comes after the taxonomic selector. Having it move if you choose a cohort is not nice.